collaboration/service/v1alpha/user_group.proto

```diff
@@ -10,11 +10,15 @@ import "google/protobuf/wrappers.proto";
 import "google/protobuf/empty.proto";
 import "global/selection/v1/pagination.proto";
 import "global/mutation/v1/mutation.proto";
+import "global/domain/v1/domain.proto";
 
 service UserGroupService {
   rpc ListUserGroups (ListUserGroupsRequest) returns (ListUserGroupsResponse);
   rpc SaveUserGroup (SaveUserGroupRequest) returns (SaveUserGroupResponse);
   rpc DeleteUserGroup (DeleteUserGroupRequest) returns (google.protobuf.Empty);
+
+  rpc ListUsersOfUserGroup (ListUsersOfUserGroupRequest) returns (ListUsersOfUserGroupResponse);
+  rpc SaveUsersOfUserGroup (SaveUsersOfUserGroupRequest) returns (SaveUsersOfUserGroupResponse);
```

Comment:
is this to save a list of users into a user group? if so, that means that we need always to send all the users ids even if we want to add just one more? would be nice to have an add/remove method?

Reply:
yes I know. For now, I make it consistent with the user management proto. But there is a todo there to do exactly what you're saying.